test/test-project/package.json

```
@@ -4,6 +4,7 @@
         "test": "npm run test:cleanup && npm run test:project && npm run test:global",
         "test:project": "jest --detectOpenHandles --testPathIgnorePatterns=test-global --modulePathIgnorePatterns='./__mocks__'",
         "test:global": "jest --detectOpenHandles --testPathPattern=test-global --modulePathIgnorePatterns='./__mocks__'",
+        "test:registry": "jest --detectOpenHandles --testPathPattern=test-global --modulePathIgnorePatterns='./__mocks__'",
```

I think you run wrong test here, probaby shoudl be 👇🏼 

```suggestion
        "test:registry": "jest --detectOpenHandles --testPathPattern=test-registry --modulePathIgnorePatterns='./__mocks__'",
```

---

Resolved @derberg